stellar/its.js

```diff
@@ -61,6 +227,51 @@ if (require.main === module) {
             mainProcessor(removeTrustedChain, chainName, options);
         });
 
+    program
+        .command('deploy-interchain-token <symbol> <name> <decimals> <salt> <initialSupply> ')
+        .description('deploy interchain token')
+        .action((symbol, name, decimal, salt, initialSupply, options) => {
+            mainProcessor(deployInterchainToken, [symbol, name, decimal, salt, initialSupply], options);
+        });
+
+    program
+        .command('deploy-remote-interchain-token <salt> <destinationChain> <gasTokenAddress> <gasFeeAmount>')
+        .description('deploy remote interchain token')
+        .action((salt, destinationChain, gasTokenAddress, gasFeeAmount, options) => {
+            mainProcessor(deployRemoteInterchainToken, [salt, destinationChain, gasTokenAddress, gasFeeAmount], options);
+        });
+
+    program
+        .command('register-canonical-token <tokenAddress>')
+        .description('register canonical token')
+        .action((tokenAddress, options) => {
+            mainProcessor(registerCanonicalToken, [tokenAddress], options);
+        });
+
+    program
+        .command(
+            'deploy-remote-canonical-token <tokenAddress> <tokenId> <tokenName> <tokenSymbol> <decimals> <destinationChain> <gasTokenAddress> <gasFeeAmount>',
```

Comment:
think about UX here as well, user shouldn't need to provide token address, name, symbol etc. Even if it's needed for auth, it should be queried on-chain via the token id instead of just moving the problem to the user